e2e/helpers.bash

```diff
@@ -0,0 +1,91 @@
+# vi: ft=bash
+
+uuid=""
+suuid=""
+ARTIFACTS_DIR=artifacts
+NAMESPACE=benchmark-operator
+
+
+basic_setup() {
+  export PROMETHEUS_TOKEN=$(oc sa get-token -n openshift-monitoring prometheus-k8s)
+  export ES_SERVER=https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com
```

Comment:
Do we want to hard-set these?

Reply:
Good catch, I have added a new mechanism to customize ES_SERVER by exporting it
The docs now say:

_A preconfigured Elasticsearch endpoint is set by default in the file helpers.bash, however it's possible to set a custom one by exporting the variable ES_SERVER._